CHANGELOG.txt

```diff
@@ -4,6 +4,10 @@ master
 Changes:
  - obspy.clients.filesystem:
    * add get_waveforms_bulk() method to SDS client (see #2616, #2626)
+ - obspy.io.reftek:
+   * store samples per second as a float (see #2678)
+   * allow for packets that are not filling the fixed amount of data space in
+     the packet when upacking data with '16' and '32' encodings (see #2678)
```

Comment:
16 bit could be read before, but it would've been wrong with packets not using the max amount of samples in it. Reading mass position channels could be mentioned in the above "store as float" part.

Reply:
Oh, right, sorry, reading through too many diffs. I think it should mentino mass positions and other streams with sample rate < 1.